keel-core/src/test/kotlin/com/netflix/spinnaker/keel/actuation/EnvironmentPromotionCheckerTests.kt

```diff
@@ -0,0 +1,567 @@
+package com.netflix.spinnaker.keel.actuation
```

Comment:
Re-adding this file was my mistake. I got confused with the conflict. Will remove.